be/src/common/config.cpp

```diff
@@ -241,6 +241,7 @@ DEFINE_Validator(doris_scanner_thread_pool_thread_num, [](const int config) -> b
     }
     return true;
 });
+DEFINE_Int32(remote_split_source_batch_size, "1024");
```

Comment:
Rethink this default value. if it is too large, all ranges may be fetched by one instance, and other instances may have no range to run

Reply:
Scan ranges have been assigned before fetching.